app.js

```diff
@@ -8,6 +8,23 @@ app.get('/api/topics', getAllTopics)
 
 app.get('/api', getAllEndpoints)
 
+app.get('/api/articles/:article_id', getArticleById)
+
+app.use((err, req, res, next) => {
+    console.log(err)
```

Comment:
Remember to remove console.logs once you're done debugging - the PRs that make it to main should be a clean version of the code